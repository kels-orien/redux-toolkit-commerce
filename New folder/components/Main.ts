import styled, {css} from 'styled-components'
import { mobileMax } from  '../utils/media'

const Main = styled.div`
    margin: 0 auto;
    padding-top: 4rem;

    ${mobileMax(css`
    padding-top: 7.5rem;
    `)};

`

export default Main