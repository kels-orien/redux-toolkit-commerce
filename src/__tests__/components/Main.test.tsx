import React from 'react'
import renderer from 'react-test-renderer'
import Main from '../../components/Main'

test('Main render correctly', () => {
    const tree = renderer.create(<Main/>).toJSON();

    expect(tree).toMatchSnapshot()

})