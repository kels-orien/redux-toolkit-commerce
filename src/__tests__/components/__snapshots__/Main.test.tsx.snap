// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main render correctly 1`] = `
<div
  className="sc-bdnxRM dXKYll"
/>
`;
